src/EventRegistry.h

```diff
@@ -60,6 +60,8 @@ class EventRegistry
 
 	void PrintDebug();
 
+	void GenerateAllHandlers();
```

Comment:
Please document what this does.

Reply:
Also, maybe rename to `ActivateAllHandlers`? (Because it's not the handlers being generated, it's the events).